Libraries/CO.CDP.Tenant.WebApiClient.Tests/TenantClientIntegrationTest.cs

```diff
@@ -0,0 +1,33 @@
+namespace CO.CDP.Tenant.WebApiClient.Tests;
+
+public class TenantClientIntegrationTest
+{
+    [Fact(Skip = "The test requires the tenant service to run.")]
```

Comment:
This test is skipped until we figure out what's the best way to run the dependent service (possibly with testcontainers).